### ISSUE TITLE
Has anyone had GTAV ban issues so far?

### ISSUE THREAD

**Issue by SdoggaMan** [antonpup/Aurora#1118]
### What are you experiencing an issue with?

- [ ] Aurora
- [x ] Aurora supported game,GTA Online
- [ ] Aurora Wrapper
- [ ] Aurora Profile
- [ ] Aurora supported device, (Fill in problematic devices)
- [ ] Aurora Scripts
- [ ] Aurora Updater
- [ ] Aurora Lighting issue
- [ ] Other, (Fill in another issue here)

### What kind of an issue is it?

- [ ] Crash
- [ ] Performance problems
- [ ] Visual bug
- [ ] Localization issue
- [ ] Enhancement suggestion
- [ ] Device issue
- [ x] Other, Aurora integration causing game code violation resulting in ban

### Expected behavior:
GTA Online shouldn't trigger a ban. After a lengthy discussion with R* telling me their ban system was perfect, which I don't believe, I wonder whether Aurora had an effect on it. 

### Actual behavior:
Played GTA for over three hundred hours just fine, but was banned after two games with modders with both ended in bugs within GTA forcing me to alt+f4 it, hence why I doubt it was Aurora. 

### How often does this problem occur?

This problem occurs... Once so far. 

### Please describe the steps to reproduce this issue:
The issue can be reproduced by doing following steps:

1. Use Aurora
2. Play GTAO
3. Get banned

### What is the version of Aurora you are experiencing this issue with?

My version is... Beta v0.6.0b

### Does the previous version of Aurora have this issue?
Since this has happened once in a year/years worth of play time it can only be a tentative 'no'.

### Please include any logs, screenshots, or files that are related to this issue:

Sorry, nothing I can really provide. Thanks for the help!

**Comment by simon-wh** [comment]
I'm not aware of anyone getting banned for using Aurora with GTA 5, I've used it many times without any issues. So I feel like it may be what you suspect, where you got banned for those games you were in.

**Comment by diogotr7** [comment]
Yeah I've never played GTA:O without Aurora and I haven't been banned.

**Comment by SdoggaMan** [comment]
Hello there!

Apologies for the long reply, after a lengthy discussion with R* I posted
this right before I went to bed.

Now I can't remember if I installed Aurora as, or after, I was finishing
the story, but it certainly wasn't far of it. I have nearly four hundred
hours played which might be nothing for most but is more than enough to
show that one ban in that time is not an Aurora thing.

Still, R* urged I believe that their system is flawless and designed to be
fairer than square, so even though I am the definitive proof otherwise, I'm
exploring the possibilities regardless. At this stage, Aurora is my only
possible culprit apart from Discord, which has nothing to do with a game
except recognizing it and displaying it's name. Some illegal modder I am..

**Comment by simon-wh** [comment]
@SdoggaMan But you said that you thought that it was likely the games you were in with modders right at the time you were banned?

**Comment by SdoggaMan** [comment]
Considering that there was a wave pf mod menus that could ban players
without the system's intervention, and that I was running from a modder
both times I had the game break and force me to kill it, yes. The first
time, the game froze in the skycam - I could still see notification updates
as players lef but otgerwise couldn't even hit escape - amd the second time
the game broke as I tried to sell a car and spawned me into the vehicle
from the warehouse with no controllability at all. Only escape responded
and it wouldn't let me leave the server, hence another alt+f4.

I don't know if the game's files broke or if I was mod-banned or what, but
I have encountered both bugs before and been fine afterwards.

**Comment by simon-wh** [comment]
How long have you been using Aurora with GTA? If you've been using it for a while like lots of other people who haven't been banned, then I highly doubt aurora was the reason you got banned

**Comment by SdoggaMan** [comment]
Yeah, like I said I've been using it for a generous amount of my playtime,
perhaps not until about when I finished story and went to Online, but for
many hours at least. That and the fact that these bugs have happened
multiple times before - while Aurora is installed, too - makes me almost
sure that it was the modders that got me banned. I mean come on, these
menus forcefully inject code, how is that not something R* will
acknowledge? It's foolish to blindly assume your game's system is perfect
amd cannot be augmented or altered unintentionally.
